sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -316,6 +316,8 @@ object FunctionRegistry {
     expression[EveryAgg]("every"),
     expression[AnyAgg]("any"),
     expression[SomeAgg]("some"),
+    expression[BoolAnd]("bool_and"),
```

Comment:
is `bool_and` just an alias of `every`? If so we can do `expression[EveryAgg]("bool_and")`

Reply:
Yes, it is. https://github.com/apache/spark/pull/26126/files#diff-fdccd9945d709da6c561b67a2e46a0d8R296, is there a way to keep the bool_and string in the schema?

Reply:
We don't need to keep the function alias in the schema. There are many examples in `FunctionRegistry`, that a function has more than one name and we always use one name when displaying.

Reply:
I see. Then `any`, `some` and `bool_or` are mathematically equivalent, I guess we can only left `AnyAgg ` and remove other implemtations